.changeset/blue-tables-sparkle.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-home': minor
+'example-app': patch
```

Comment:
This changeset is not needed, since we are releasing a new package. And `example-app` doesn't get released. :)

Reply:
Ah, great input! Fixed in 4ea6e81!